apps/mark-scan/backend/src/app.test.ts

```diff
@@ -477,92 +475,58 @@ async function mockLoadFlow(
   testApiClient: grout.Client<Api>,
   testDriver: MockPaperHandlerDriver
 ) {
+  testDriver.setMockPaperContents(BLANK_PAGE_IMAGE_DATA);
   await testApiClient.setAcceptingPaperState({
     paperTypes: ACCEPTED_PAPER_TYPES,
   });
   await waitForStatus('accepting_paper');
 
   testDriver.setMockStatus('paperInserted');
   clock.increment(delays.DELAY_PAPER_HANDLER_STATUS_POLLING_INTERVAL_MS);
-  await waitForStatus('loading_paper');
+  await waitForStatus('loading_new_sheet');
+
+  clock.increment(delays.DELAY_AUTH_STATUS_POLLING_INTERVAL_MS);
+  await waitForStatus('waiting_for_voter_auth');
 
   mockCardlessVoterAuth(mockAuth);
   clock.increment(delays.DELAY_AUTH_STATUS_POLLING_INTERVAL_MS);
   await waitForStatus('waiting_for_ballot_data');
 }
 
-async function mockLoadAndPrint(
-  testApiClient: grout.Client<Api>,
-  testDriver: MockPaperHandlerDriver
-) {
-  await mockLoadFlow(testApiClient, testDriver);
-  await testApiClient.printBallot({
-    languageCode: 'en',
-    precinctId: '21',
-    ballotStyleId: '12' as BallotStyleId,
-    votes: {},
-  });
-  await waitForStatus('presenting_ballot');
-}
-
 test('ballot invalidation flow', async () => {
```

Comment:
Logic's already tested in `state_machine.test` - simplifying to just test the glue code.